src/Interpreters/SubqueryForSet.cpp

```diff
@@ -8,13 +8,19 @@
 namespace DB
 {
 
+SubqueryForSet::SubqueryForSet() = default;
+SubqueryForSet::~SubqueryForSet() = default;
+SubqueryForSet::SubqueryForSet(SubqueryForSet &&) = default;
+SubqueryForSet & SubqueryForSet::operator= (SubqueryForSet &&) = default;
```

Comment:
Why do you place them here? What would go wrong if they would be defined in class?

Reply:
Only `~SubqueryForSet()` must be implemented here cause `QueryPlan` is incomplete in header, and `~std::unique_ptr<QueryPlan>`  not always can be implemented. Everything else is here is just for consistency.